research/slim/train_image_classifier.py

```diff
@@ -340,12 +340,10 @@ def _get_init_fn():
   # TODO(sguada) variables.filter_variables()
   variables_to_restore = []
   for var in slim.get_model_variables():
-    excluded = False
     for exclusion in exclusions:
       if var.op.name.startswith(exclusion):
-        excluded = True
         break
-    if not excluded:
+    else:
```

Comment:
This is the unmatched `else` I was referring to.

Reply:
No, it matches with **for**. It's a for...else... structure.

Reply:
Huh. TIL. That does indeed appear to be equivalent.

Let me get back to you tomorrow.

Reply:
Look forward to you.